Makefile

```diff
@@ -29,4 +29,4 @@ clean:
 	rm -rf dashboards_out prometheus_alerts.yaml prometheus_rules.yaml
 
 test: prometheus_alerts.yaml prometheus_rules.yaml
```

Comment:
Do we need to have both `prometheus_alert.yml` and `prometheus_rules.yml`? Both sets are rules (be it recording or alerting) rules. I suppose it would make sense to have separate file for each set of rules. I would just prefer it if we called both of these 'rules' in some way, i.e. `prometheus_alert_rules.yml` and `prometheus_record_rules`.

Reply:
I feel file name doesn't matter and we can have our own names defined. Will do the required changes.